sklearn/linear_model/tests/test_omp.py

```diff
@@ -33,16 +32,16 @@
 
 def test_correct_shapes():
     assert (orthogonal_mp(X, y[:, 0], n_nonzero_coefs=5).shape ==
-                 (n_features,))
+            (n_features,))
     assert (orthogonal_mp(X, y, n_nonzero_coefs=5).shape ==
-                 (n_features, 3))
+            (n_features, 3))
 
 
 def test_correct_shapes_gram():
     assert (orthogonal_mp_gram(G, Xy[:, 0], n_nonzero_coefs=5).shape ==
-                 (n_features,))
+            (n_features,))
     assert (orthogonal_mp_gram(G, Xy, n_nonzero_coefs=5).shape ==
-                 (n_features, 3))
```

Comment:
It's ok for this time, but next time please try to refrain to change the formatting of the lines that are fare away from the real changes of your PR to keep the review focused on the main topic (and keep the git history meaningful).

Reply:
I apologize, I have a lot to learn and I appreciate your time and effort. Thank you for the valuable feedback